### ISSUE TITLE
VariantValidator query broken

### ISSUE THREAD

**Issue by stolpeo** [bihealth/varfish-server#523]
**Describe the bug**
Clicking on VariantValidator Query results in a gray overlay that makes the page non-interactable.

**To Reproduce**
1. Go to filter page and create filter results.
2. Click on dropdown beneath IGV button and select "VariantValidator query"
3. See error

**Expected behavior**
Modal showing the query results that is close-able.

**Comment by stolpeo** [comment]
#156

**Comment by holtgrewe** [comment]
**Root Cause Analysis (Continued)**

`git bisect` yields:

```
 git bisect bad
4a9b162dd23ba3159a35706325f0caf61f8daaba is the first bad commit
commit 4a9b162dd23ba3159a35706325f0caf61f8daaba
Author: Manuel Holtgrewe <manuel.holtgrewe@bihealth.de>
Date:   Fri May 6 13:54:25 2022 +0200

    Adding "variant carrier" popup modal (#470).

    Related-Issue: #470
    Closes: #470
    Projected-Results-Impact: none

 HISTORY.rst                                        |  2 +
 varfish/static/js/variant_carrier.js               | 17 ++++++
 variants/templates/variants/filter_result/row.html | 27 ++++++++++
 .../filter_result/variant_carriers_result.html     | 62 ++++++++++++++++++++++
 variants/templates/variants/scripts.html           |  2 +
 variants/templatetags/variants_tags.py             | 15 ++++++
 variants/tests/test_views.py                       | 47 ++++++++++++++++
 variants/urls.py                                   |  5 ++
 variants/views.py                                  | 47 ++++++++++++++++
 9 files changed, 224 insertions(+)
 create mode 100644 varfish/static/js/variant_carrier.js
 create mode 100644 variants/templates/variants/filter_result/variant_carriers_result.html
```

**Comment by stolpeo** [comment]
@holtgrewe thanks, I'm currently looking into this
